invenio_previewer/templates/semantic-ui/invenio_previewer/txt.html

```diff
@@ -0,0 +1,22 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2021 Northwestern University.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- extends config.PREVIEWER_ABSTRACT_TEMPLATE %}
+
+{% block panel %}
+<div class="ui container">
+    <div class="ui grid column">
+        <div class="column">
+            <pre>
+              {{ content | sanitize_html() | safe }}
```

Comment:
Shouldn't this just be ``content``?

Reply:
We shouldn't trust what is inside `content` no? (couldn't they otherwise inject things?)

Reply:
If we use just ``{{ content }}`` then Jinja will do auto-escaping and ensure that no harmful content ...i.e. the page will simply display whatever the characeters are.......the ``sanitize_html | safe`` will first remove dangerous html and then render as html.

E.g. imaging you have a text file containing this:

```
</pre><a href="evilsite.com">Buy this</a>
```

Because it's plain text, you want to render it as above. However, since both ``pre`` and ``a`` are white-listed tags, then you'll close the ``pre``-tag in the template, and the link will be rendered instead.

Reply:
ok I was planning on double-checking just that case and that makes sense.

Reply:
Made the change and it's good now:

![image](https://user-images.githubusercontent.com/1932651/127494666-77b632ec-b7b5-4faa-881e-3c45f3749ef2.png)